src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Serialization/ProjectSnapshotHandleJsonConverter.cs

```diff
@@ -4,11 +4,13 @@
 using System;
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+using Microsoft.CodeAnalysis.Razor.Serialization;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor.Serialization
 {
+    [JsonConverter(typeof(ProjectSnapshotHandleJsonConverter))]
```

Comment:
I suspect this wasn't intended to go here since we don't need to JsonConvert our JsonConverter?

Reply:
Nope, and removed with the latest commit :)

This is some recursive serialization 😆